examples/hvn-peering/nodejs/index.ts

```diff
@@ -0,0 +1,80 @@
+import * as pulumi from "@pulumi/pulumi";
+import * as hcp from "@grapl/pulumi-hcp";
+import * as aws from "@pulumi/aws";
+
+const aws_tags = {
+    "pulumi:project": pulumi.getProject(),
+    "pulumi:stack": pulumi.getStack()
+}
+
+const hvn = new hcp.Hvn(
+    `${ pulumi.getProject() }-hvn`,
```

Comment:
NITPICK AND POSSIBLY NOT WORTH FOLLOWING UP ON: 
I believe the spaces here in this template-literal is not required. Might be worth throwing a typescript-linter on or something

Reply:
Not a bad idea... I'll make a TODO.